Tests/SwiftCrossUITests/SwiftCrossUIBackendTests.swift

```diff
@@ -0,0 +1,158 @@
+import Testing
+
+@testable import SwiftCrossUI
+
+#if canImport(AppKitBackend)
+    @testable import AppKitBackend
+#elseif canImport(DefaultBackend)
+    // Is this close enough to a "Mock Backend"?
+    @testable import DefaultBackend
+#endif
+
+// TODO: Create mock backend so that this can be tested on all platforms. There's
+//       nothing AppKit-specific about it.
+@Suite(
+    "Testing for Graphical Backends",
+    .tags(.Backend)
+)
+struct BackendTests {
+    enum BackendTestError: Error {
+        case FailedBitmapBack
+        case FailedTiffRep
+
+        var errorDescription: String? {
+            switch self {
+                case .FailedBitmapBack: "Failed to create bitmap backing"
+                case .FailedTiffRep: "Failed to create tiff representation"
+            }
+        }
+    }
+
+    #if canImport(AppKitBackend)
+        @Test(
+            "Validates Observation will not give up the thread entirely when sleeping the thread",
+            .tags(.Observation)
+        )
+        func ThrottledStateObservation() async {
+            class MyState: SwiftCrossUI.ObservableObject {
+                @SwiftCrossUI.Published
+                var count = 0
+            }
+
+            /// A thread-safe count.
+            actor Count {
+                var count = 0
+
+                func update(_ action: (Int) -> Int) {
+                    count = action(count)
+                }
+            }
+
+            // Number of mutations to perform
+            let mutationCount = 20
+            // Length of each fake state update
+            let updateDuration = 0.02
+            // Delay between observation-causing state mutations
+            let mutationGap = 0.01
+
+            let state = MyState()
+            let updateCount = Count()
+
+            let backend = await AppKitBackend()
+            let cancellable = state.didChange.observeAsUIUpdater(backend: backend) {
+                Task {
+                    await updateCount.update { $0 + 1 }
+                }
+                // Simulate an update of duration `updateDuration` seconds
+                Thread.sleep(forTimeInterval: updateDuration)
+            }
+            _ = cancellable  // Silence warning about cancellable being unused
+
+            let start = ProcessInfo.processInfo.systemUptime
+            for _ in 0..<mutationCount {
+                state.count += 1
+                try? await Task.sleep(for: .seconds(mutationGap))
+            }
+            let elapsed = ProcessInfo.processInfo.systemUptime - start
+
+            // Compute percentage of main thread's time taken up by updates.
+            let ratio = Double(await updateCount.count) * updateDuration / elapsed
+            #expect(
+                ratio <= 0.85,
+                """
+                Expected throttled updates to take under 85% of the main \
+                thread's time. Took \(Int(ratio * 100))%
+
+                Duration: \(elapsed), \(start) <=> \(end)
```

Comment:
Remove the `\(start) <=> \(end)` part. They'll just be big system uptime numbers that don't provide any more information than `\(elapsed)`.

Add `seconds` unit to duration text.